src/App.tsx

```diff
@@ -1,22 +1,19 @@
 import React from 'react';
+import { Routes, Route } from 'react-router-dom';
 import './App.css';
+import { Home } from './pages/Home';
+import { About } from './pages/About';
 
 const App = (): JSX.Element => (
-  <div className="App">
-    <header className="App-header">
-      <p>
-        Edit <code>src/App.tsx</code> and save to reload.
-      </p>
-      <a
-        className="App-link"
-        href="https://reactjs.org"
-        rel="noopener noreferrer"
-        target="_blank"
-      >
-        Learn React
-      </a>
-    </header>
-  </div>
+  <Routes>
+    <Route 
+      element={<Home />}
+      path='*' 
```

Comment:
For this one, I suggest you to keep the `'/'` for the `Home` page, and you can create a `NotFoundPage` linked to `*` which will catch any other url.
For a user, it's easier to understand that he typed a wrong page url when he lands on a "Page not found" than the landing page.

Reply:
I resolved it in the second commit 510baf84c8fff1ddf50a841f3d804cc2f733c12e, maybe you haven't seen it ? 😊